Dockerfile

```diff
@@ -25,10 +27,10 @@ RUN groupadd -g 999 tester && \
 USER tester
 
 # Install terminus
-RUN git clone https://github.com/pantheon-systems/terminus.git ~/terminus
-RUN cd ~/terminus && git checkout 3.x && composer install
-# Terminus 3 binary current is named t3.
-RUN ln -s ~/terminus/bin/t3 /usr/local/bin/terminus
+RUN curl -L https://github.com/pantheon-systems/terminus/releases/download/3.1.1/terminus.phar -o terminus.phar && \
```

Comment:
`curl -LO https://github.com/pantheon-systems/terminus/releases/download/3.1.1/terminus.phar` does the same thing.